InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -2284,7 +2284,7 @@ private string ConditionalTextureNameOrUUID(SceneObjectPart part, UUID assetID)
 
             // We need to provide the name if present in the prim.
             string result = InventoryName(assetID);
-            if (result != "")
+            if (result != String.Empty)
```

Comment:
I missed one!

Analysis: if this was `!String.IsNullorEmpty()` then there would be a semantic change, but only if `InventoryName` pulled up an asset with a null name.  Not sure that's possible.  Either way I'm sure it's a better test as I'm not sure `llGetPrimParams` should ever have a null name returned....  (`LSLSystemAPI::GetPrimParams()` is a primary consumer of this method.)